crates/storage/Cargo.toml

```diff
@@ -15,10 +15,15 @@ indexmap.workspace = true
 parking_lot.workspace = true
 sha2.workspace = true
 thiserror.workspace = true
-uuid.workspace = true
 
 calimero-store = { path = "../store", features = ["datatypes"] }
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+uuid = { version = "1.10.0", features = ["serde", "v5"] }
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+uuid = { version = "1.10.0", features = ["serde", "v4"] }
+
```

Comment:
^

Reply:
move to `[dependencies]`, unless you actually need that `v5` feature activated, in which case, I can't find any examples of it's use

Reply:
As per above, they are not the same - the WASM cannot have the v4.

Reply:
Replied above, it doesn't need to, yet this still stands